Rachis/Rachis/RaftEngineStatistics.cs

```diff
@@ -68,6 +73,16 @@ public void ReportCommitIndex(long index)
         }
     }
 
+    public class FollowerLastSentEntries
+    {
+        public FollowerLastSentEntries(ConcurrentDictionary<string, long> f2e)
+        {
+            FollowersToLastSent = f2e;
+        }
+        public readonly ConcurrentDictionary<string,long> FollowersToLastSent;
```

Comment:
This is probably not doing what you expect it to be. You are holding on to the _mutating_ value here.
Is this intended as a snapshot or just to show what is _now_?